api_tester/lib/api_tests/models/customer_info_wrapper_api_test.dart

```diff
@@ -7,10 +7,6 @@ class _CustomerInfoApiTest {
     CustomerInfo customerInfo = CustomerInfo.fromJson(json);
   }
 
-  void _checkToJson(CustomerInfo customerInfo) {
-    Map<String, dynamic> json = customerInfo.toJson();
```

Comment:
This is one of the breaking changes. I didn't think we need to keep the `toJson` method in all our models.